evm/update-static-gas-options.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const chalk = require('chalk');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    utils: { parseUnits },
+} = ethers;
+const readlineSync = require('readline-sync');
+
+const { printError, printInfo, printObj, saveConfig } = require('./utils');
+
+async function updateStaticGasOptions(chain, options, filePath) {
+    const { rpcUrl, yes } = options;
+    const provider = getDefaultProvider(rpcUrl);
+    const network = await provider.getNetwork();
+
+    if (!yes) {
+        const anwser = readlineSync.question(
+            `Proceed with the static gasOption update on network ${chalk.green(network.name)} with chainId ${chalk.green(
+                network.chainId,
+            )} ${chalk.green('(y/n)')} `,
+        );
+        if (anwser !== 'y') return;
+    }
+
+    try {
+        const gasPrice = parseUnits((await provider.getGasPrice()).toString(), 'gwei') * 5;
+        const staticGasOptions = {};
+        staticGasOptions.gasLimit = 130000;
+        staticGasOptions.gasPrice = gasPrice;
+        chain.staticGasOptions = staticGasOptions;
+        printInfo(`GasOptions updated succesfully and stored in config file ${filePath}`);
```

Comment:
```suggestion
        chain.staticGasOptions.gasLimit = chain.staticGasOptions.gasLimit || 3e6;
        chain.staticGasOptions.gasPrice = gasPrice;
        printInfo(`GasOptions updated succesfully and stored in config file ${filePath}`);
```

we can keep a high default

but if it's already set, then we shouldn't override. some chains have different gas metering.